soh/soh/Enhancements/randomizer/3drando/logic.cpp

```diff
@@ -194,13 +194,11 @@ namespace Logic {
   bool BombchuDrop      = false;
   bool AmmoCanDrop      = false;
 
-  bool BuyBombchus10    = false;
-  bool BuyBombchus20    = false;
+  bool BuyBombchus      = false;
```

Comment:
It may have been at some point if it was assumed based on wallet, but the logic currently checks the item being sold itself for the price, which is a good thing as it potentially allows settings to change it, making the distinction pointless.

Reply:
Good enough for me